test/integration/targets/delegate_to/test_delegate_to.yml

```diff
@@ -1,9 +1,11 @@
 - hosts: testhost3
```

Comment:
Think a sane test might be to ICMP ping the IPs (eg, would that work on the platforms you've tried running this test on)?